collection/largest.ts

```diff
@@ -17,12 +17,12 @@ export function largest<P extends string, T extends { [K in P]: number }>(
   property: P,
   items: Iterable<T>,
 ): T[];
-export function largest(
+export function largest<T>(
```

Comment:
These `T`'s are meaningless in this signature. We should find an alternative before merging.

Reply:
Addressed.